Admin.js

```diff
@@ -11,7 +11,7 @@
 
 var Admin = {
```

Comment:
- eslint: All 'var' declarations must be at the top of the function scope.
- eslint: Combine this with the previous 'var' statement.
- eslint: Unexpected var, use let or const instead.
- eslint: Unary operator '++' used.
- eslint: Strings must use doublequote.